apps/documentation/src/content/docs/architecture.md

```
@@ -0,0 +1,1856 @@
+    tools,
+    maxSteps: 5  // Allow multiple tool calls
+  });
+  
+  // Return readable stream
+  return result.toDataStreamResponse();
+}
+```
+
+**Streaming Flow:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)